python/mxnet/_numpy_op_doc.py

```diff
@@ -22,37 +22,73 @@
 
 def _np_reshape(a, newshape, order='C'):
     """
-    reshape(a, newshape, order='C')
-
     Gives a new shape to an array without changing its data.
 
     Parameters
     ----------
     a : ndarray
         Array to be reshaped.
-    newshape : int or tuple of ints
+    newshape : int, tuple of ints or list of ints
         The new shape should be compatible with the original shape. If
         an integer, then the result will be a 1-D array of that length.
         One shape dimension can be -1. In this case, the value is
         inferred from the length of the array and remaining dimensions.
-    order : {'C'}, optional
+    order : 'C', optional
         Read the elements of `a` using this index order, and place the
         elements into the reshaped array using this index order.  'C'
         means to read / write the elements using C-like index order,
         with the last axis index changing fastest, back to the first
-        axis index changing slowest. Other order types such as 'F'/'A'
-        may be added in the future.
+        axis index changing slowest. We will add 'F'/'A' order types
+        in the future.
 
     Returns
     -------
     reshaped_array : ndarray
-        It will be always a copy of the original array. This behavior is different
-        from the official NumPy package where views of the original array may be
-        generated.
+        This will be a new view object if possible; otherwise, it will
+        be a copy.  Note there is no guarantee of the *memory layout* (C- or
+        Fortran- contiguous) of the returned array.
 
-    See Also
+
+    Notes
+    -----
+    It is not always possible to change the shape of an array without
+    copying the data. If you want an error to be raised when the data is copied,
+    you should assign the new shape to the shape attribute of the array::
+
+     >>> a = np.zeros((10, 2))
+     # A transpose makes the array non-contiguous
+     >>> b = a.T
+     # Taking a view makes it possible to modify the shape without modifying
```

Comment:
Seems like you're going to show an example of "taking a view"... but then it isn't clear to me in the example what a view is. Is this comment associated with line 60 or 64?